src/main/java/uk/ac/ebi/eva/pipeline/model/Csq.java

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.model;
+
+import org.beanio.annotation.Field;
+import org.beanio.annotation.Record;
+
+/**
+ * Models the CSQ field generated by VEP. See 'VCF output' in http://www.ensembl.org/info/docs/tools/vep/vep_formats.html
+ */
+@Record
+public class Csq {
```

Comment:
I understand the fields inside CSQ are configurable, would this implementation support that use case?